Dialogue/Afin/np_010030.csv

```diff
@@ -19,70 +19,70 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
+11_voice_NPC_5003_105#1,"""We can change classes but the higher ranks have already chosen theirs. Why did it change...?"""
+11_voice_NPC_5003_104#2,"""There are some ARKS that are not only good at fighting but gathering info. I wonder if I'm the same..."""
+11_voice_NPC_5003_104#3,"""I knew finding people would be hard but this will take a while. Guess I'll just have to be patient."""
+11_voice_NPC_5003_109#0,"""More Darkers appeared! At least that's what it feels like a lot these days. Maybe it's just me?"""
+11_voice_NPC_5003_101#2,"""Seeing you complete so many client orders is making me happy too. Hehe."""
+11_voice_NPC_5003_108#1,"""I was told by my superiors that “you don't have any courage but you know that”. A bit harsh don't you think?"""
+Afin_15_020#0,"""The thing that's scary is that there are people who think they have courage when they don't. ...Guess they're just reckless."""
+11_voice_NPC_5003_105#2,"""I've been thinking. The Darkers have been our enemies for a while now but It hasn't been properly been recorded in history..."""
+11_voice_NPC_5003_104#4,"""ARKS are...loved by everyone. But I never thought it would be this tiring...ha-."""
+11_voice_NPC_5003_100#3,"""The best ARKS...the Council of Six?  I guess. They're all pretty strong."""
 Afin_18_020#0,"""……なーんて、案外その辺りを<br>ふつーに歩いてたりしてな。<br>もう会っちゃってたり、とか？"""
-11_voice_NPC_5003_109#1,"""直接アークスシップが襲われたのに<br>みんな慌ててないって不思議だよな。<br>よくあること、とか言われる始末。"""
-Afin_19_020#0,"""……それを普通と思っちまうのって<br>なんかいろいろおかしくなってる証拠の<br>ような気がするんだよなあ。"""
-11_voice_NPC_5003_104#5,"""おれは別に<br>誰かを守りたいって気持ちで<br>アークスになったわけじゃないけど……"""
-Afin_20_020#0,"""やっぱ、ここを攻められたりすると<br>守りたいって自然と思うんだな。<br>なんか、自分自身がすっげえ不思議。"""
-11_voice_NPC_03_104#0,"""ダークファルスが復活したからか<br>アークス全体の空気が重いよなぁ。"""
-Afin_mb9_020#0,"""まあ、当たり前っちゃ当たり前だけど<br>へこんでばっかいても仕方がないと<br>おれは思うぜ。"""
-11_voice_NPC_03_106#0,"""ウワサの暴走龍ってさ<br>なんか結構数いるみたいだな。"""
-Afin_mb10_020#0,"""べらぼーに強いヤツもいるらしいから<br>気をつけた方がいいぜ。"""
-11_voice_NPC_03_104#1,"""まー、戦技大会にせよ何にせよ<br>騒いでうっぷんを晴らすってのは<br>いいことだと思うぜ。"""
-Afin_mb11_020#0,"""まわりでも結構参加する人が<br>多いみたいだし……<br>おれはどうしようかなぁ。"""
-11_voice_NPC_5003_105#3,"""…………<br>あっ、わ、悪い相棒。<br>ちょっと考え事してた。"""
-Afin_mb12_020#0,"""考えても何も解決しないって<br>そんなことわかってるんだけど……<br>おれ、切り替えが下手でさ。"""
-11_voice_NPC_5003_105#4,"""…………<br>あっ、わ、悪い相棒。<br>ちょっと考え事してた。"""
-Afin_mb13_020#0,"""考えても何も解決しないって<br>そんなことわかってるんだけど……<br>おれ、切り替えが下手でさ。"""
-11_voice_NPC_5003_104#6,"""惑星リリーパの採掘場跡、か。<br>……何かありそうな気がする。"""
+11_voice_NPC_5003_109#1,"""It's weird how the ARKS ship can be attacked directly and no one panics. I guess it happens too often for people to get upset."""
+Afin_19_020#0,"""...I feel like there is something wrong with you if you think that that is normal...。"""
+11_voice_NPC_5003_104#5,"""I didn't join ARKS because I wanted to protect someone."""
+Afin_20_020#0,"""Like I thought, if we get attacked here I end up thinking I want to protect everyone. It's weird to think about."""
+11_voice_NPC_03_104#0,"""Ever since the Dark Falz has been revived, everyone's been on edge."""
```

Comment:
'...the Dark Falz' or just 'since Dark Falz'?

Reply:
There is a misspelling:
 seperated or separated?

Reply:
since Dark Falz